ddp-workspace/tslint.json

```diff
@@ -183,18 +183,16 @@
     "directive-selector": [
       true,
       "attribute",
-      "app",
       "camelCase"
     ],
     "component-selector": [
       true,
       "element",
-      "app",
```

Comment:
the 'components selectors need to start with `app` prefix' was annoying long enough. Same for directives.